src/conn/conn_api.c

```diff
@@ -2601,6 +2591,18 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
     WT_ERR(__wt_config_gets(session, cfg, "operation_timeout_ms", &cval));
     conn->operation_timeout_us = (uint64_t)(cval.val * WT_THOUSAND);
 
+    /*
+     * Set compatibility versions early so that any subsystem sees it. Call after we own the
```

Comment:
"sees it" should be "sees them"

Reply:
Fixed. The comment was actually outdated.